t/t3002-pmi.t

```diff
@@ -206,7 +206,7 @@ test_expect_success 'flux-pmi --libpmi-noflux fails w/ flux libpmi.so' '
 	    flux pmi --method=libpmi:$(cat libpmi) --libpmi-noflux barrier"
 '
 test_expect_success 'flux broker refuses the Flux libpmi.so and goes single' '
-	FLUX_PMI_DEBUG=1 \
+	FLUX_PMI_DEBUG=1 FLUX_PMI_CLIENT_METHODS="libpmi single" \
```

Comment:
I'm unclear on why the test should _not_ ensure that both libpmi2 and libpmi plugins refuse to boot from Flux's own library. Maybe the commit message just needs to explain the why of the problem statement. (Apologies if it should be obvious)

Reply:
It could check both, but it says its checking libpmi and I add another check for libpmi2 later on, so it was more just an effort to organize tests by plugin and not have the behavior of one plugin affect a test that claims to be checking the other. I'll update the commit message to try to be more clear.

Reply:
Ah! That makes sense (it should have been obvious, sorry!)